endpoints.json

```diff
@@ -19,6 +19,13 @@
       "topics": [{ "slug": "football", "description": "Footie!" }]
     }
   },
+  "GET /api/users": {
+    "description": "serves an array of all users",
+    "queries": [],
+    "exampleResponse": {
+      "users": [{ "username": "bob1", "name": "robert!" , "avatar_url": "url" }]
```

Comment:
do we want users to have ! in their 'name'? If not, we should give an ideal example - 'robert'